cloudformation/watched-account.template.yaml

```diff
@@ -50,18 +38,6 @@ Resources:
             Resource: "*"
             Action:
             - s3:GetEncryptionConfiguration
-            # Analyse security groups
```

Comment:
These changes are to the role SHQ assumes for watching other AWS accounts. This is separate to the Lambdas that run in each account to alert on configuration changes.

I think this ought to be a valid change now that we've removed the dashboards that required these permissions, but I think I'd want to double check that the actual services behind those dashboards in SHQ have been disabled. At first glance this doesn't seem to be the case, which means that SHQ would be trying (and failing) to read this data after it starts.

My suggestion would be for us to leave this policy as-is here, and tackle a wider change that removes deprecated functionality from SHQ (with a corresponding reducuction in its policy) as another piece of work.